CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Fix topic creation [#1333](https://github.com/opendatateam/udata/pull/1333)
 - Add a `udata worker status` command to list pending tasks.[breaking] The `udata worker` command is replaced by `udata worker start`. [#1324](https://github.com/opendatateam/udata/pull/1324)
+- Prevent crawlers from indexing spammy datasets and reuses [#1334](https://github.com/opendatateam/udata/pull/1334)
```

Comment:
I assume it's normal that the url given points to the pull request and not to an issue?

Reply:
Yep, when give the most relevant link to explain the point, issue or pull-request.